### PR TITLE
Waterfall Network

### PR DESCRIPTION
**Pull request by grybniak**
Waterfall is a highly scalable smart contract network for the development of various decentralized applications (DApps). The distributed protocol is based on DAG (Directed Acyclic Graph) technology, with a 2-level fast finality Proof-of-Stake (PoS) consensus protocol, in which millions of validators can participate. It integrates with the Ethereum Virtual Machine (EVM) that all Ethereum-based apps can be run on this network. Therefore, Waterfall can ensure a favorable environment for the provision and consumption of a wide spectrum of enterprise-class services for business and social activities, in a convenient format within the framework of a decentralized network.

### CODE REVIEW

**Comment by knagware9** [comment]
Project proposal looks good but we need more details what it offers which other DLTs in LFDT dont have..

**Review by alvaropicazo — APPROVED** [review]
LGTM.

**Comment by grybniak** [comment]
> LGTM.

Thank you!

File: labs/waterfall.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: Waterfall Network Proposal
+parent: Labs
+---
+# Lab Name
+[Waterfall Network](https://waterfall.network/)
+
+# Short Description
+Waterfall is a highly scalable smart contract network for the development of various decentralized applications (DApps).
```
Comment (nidhi-singh02) [inline labs/waterfall.md:10]: Thanks for the proposal. Need more clarity on the protocol, few questions I have:
1) What sort of protocol is it ? Please elaborate how does it integrate with EVM.
2) Does the smart contract transaction execution happen on its own network or the Layer 1 network ?
3) Does it support Non EVM use cases as well ?
4) The Dapps that the user develops using Waterfall can interact with which blockchain network ? Or is it like Waterfall network is the Layer one itself ?

Comment (grybniak) [reply labs/waterfall.md:10]: Thank you for the good questions: 

1. It is a layer one network with its own state and security. It uses actual EVM(not any sort of emulation), and on the level of EVM everything works identically to Ethereum - EVM receives an ordered set of transactions for execution, executes them, and the network state is updated. 

2. Everything happens on Waterfall own layer 1 network.  

3. Not at this stage, but the modularity of the virtual machine and its expansion beyond the scope of EVM were discussed and estimated, but in the team's opinion, they are not priorities as of now. 

4. Waterfall is layer one itself, but it is already connected through bridges to other EVM-based networks, s.a. Base, Polygon, Arbitrum, BSC, and later to Bitcoin and other non-EVM networks.

Comment (nidhi-singh02) [reply labs/waterfall.md:10]: Thanks a lot for the responses. Great proposal! I will approve it!

**Review by nidhi-singh02 — APPROVED** [review]

File: labs/waterfall.md (hunk 1 above)
Comment (grybniak) [reply labs/waterfall.md:10]: Thank you!

**Pull request merged**